c/include/ml-api-service.h

```diff
@@ -39,6 +39,185 @@ extern "C" {
  * @{
  */
 
+/**
+ * @brief A handle for ml-service instance.
+ * @since_tizen 7.0
+ */
+typedef void *ml_service_h;
+
+/**
+ * @brief Callbacks for the events from ml-service.
+ * @since_tizen 9.0
+ * @todo TBU (define enum for ml-service event)
+ */
+typedef struct {
+  void (*new_data) (ml_service_h handle, const char *name, const ml_tensors_data_h data, void *user_data);
+  void (*event) (ml_service_h handle, int event, void *user_data);
```

Comment:
Are you sure that event-invoker is never going to give any further information other than the event type enum?

Reply:
added new enum for ml-service event type and event data.